source/addonHandler/__init__.py

```diff
@@ -591,15 +616,7 @@ def enable(self, shouldEnable: bool) -> None:
 				# Undoing a pending disable.
 				state[AddonStateCategory.PENDING_DISABLE].discard(self.name)
 			else:
-				if self.canOverrideCompatibility and not self.overrideIncompatibility:

```

Comment:
This code was removed, since there is no way to call `enable` on an incompatible add-on. It is either compatible with the running version of NVDA, or marked as overridden compatibility, so prompting user in an impossible situation seems quite pointless.

Reply:
I think this was a remnant of needing to support add-ons manager in parallel the add-ons store.